### PR TITLE
Added Period and Weed Decks

### LINKED ISSUE #1: Add Period Pack
Released 11 July 2017 with [Cards Against Humanity For Her](https://cardsagainsthumanityforher.com/).

### LINKED ISSUE #3: Add Weed Pack
Released 11 July 2017

### PR DESCRIPTION
**Pull request by mawkee**
Closes #1 
Closes #3

### CODE REVIEW

File: src/period/white.html.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+A diverse group of female friends casually discussing the side effects of birth control.
+A woman president.
+Always&copy; Infinity Extra Heavy Overnight Pads with Wings.
+An emotionally draining friendship.
+Carrying a fetus to term.
+Catching a whiff of my vag.
+Dancing carefree in white linen pants.
+Destroying a pair of underwear.
+Drinking Beyoncé's DivaCup and becoming immortal.
```
Comment (crhallberg) [inline src/period/white.html.txt]: Only concern I have is that this acute needs to be escaped for HTML.

**Comment by crhallberg** [comment]
Thanks for doing these! Any feedback on the method or tools?

**Comment by mawkee** [comment]
I didn't really dig through all the Python code. Most I've seen is easy to understand, but I'd say more info (including documentation) would be nice.

I'm actually thinking about building a bot for CAH on Telegram (using bottery) and being able to use json-against-humanity will be really useful ;-)

**Comment by crhallberg** [comment]
That's why we're here! I'll try to remember to take a look once you get going.

**Pull request merged**